index.css

```diff
@@ -0,0 +1,6 @@
+header {
+    position: fixed;
+    width: 100%;
+    top: 0;
```

Comment:
- I would be happy to recommend you that to follow linters rule of indentation for more readability of your code

Reply:
- [ ] Comment is polite, and clear & the reason for the change given. 👏🏾